components/organisms/TitleSection.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from "prop-types";
+/**
+ *  page title section
+ */
+export default function TitleSection(props) {
+  return (
+    <div>
+      <h2 className="pb-2 font-bold">{props.title}</h2>
+      <hr className="border-red-800 border-2"></hr>
```

Comment:
border is too big compared to canada.ca,

Reply:
shoujld be fixed when i do Kris' recommend :+1: